samples/cellular/modem_shell/src/ping/icmp_ping_shell.c

```diff
@@ -165,7 +165,11 @@ int icmp_ping_shell_th(const struct shell *shell, size_t argc, char **argv,
 			}
 			break;
 			}
-		case 'h': /* help */
+
+		case '?':
+			mosh_error("Unknown option. See usage:");
```

Comment:
Because of the `if (argc < 3)` check above, this isn't printed with invalid parameters unless there's at least two of them. Maybe that check could be removed, it seems that the presence of mandatory parameter `-d` is anyway checked separately.

Reply:
Yep, removed. I need to check all these `if (argc <` comparisons.